src/lib/grammar.js

```diff
@@ -129,12 +129,13 @@ function buildRegex(map, keyEx, valEx) {
         '(?<' + keyEx + '>',
             keys,
         ')',
-        // value is optional
         '(?:\\(',
             '(?<' + valEx + '>',
                 GRAMMAR.VALUES,
             ')',
-        '\\))?',
+        '\\))',
+        // value is optional only for helper
+        keyEx === 'helper' ? '?' : ''
```

Comment:
I'd have everything to have a param by default and a flag to tell when it's optional.